src/integration-test/java/com/commercetools/project/sync/CliRunnerIT.java

```diff
@@ -100,6 +107,17 @@ static void setupSourceProjectData(@Nonnull final SphereClient sourceProjectClie
                 ResourceTypeIdsSetBuilder.of().addCategories())
             .build();
 
+    final StateDraft stateDraft =
+        StateDraftBuilder.of("State 1", StateType.PRODUCT_STATE)
+            .roles(Collections.emptySet())
+            .description(LocalizedString.ofEnglish("State 1"))
+            .name(LocalizedString.ofEnglish("State 1"))
```

Comment:
It seems that the method `.ofEnglish()` has been imported already in existing code. Therefore it is not necessary to call `LocalizedString` class explicitly.

Reply:
https://github.com/commercetools/commercetools-project-sync/pull/133/commits/42d37fe9f4c9a4b06f35ab3b9fc60aba3bb1d1fa